testgrid/tgrun/pkg/runner/embed/runcmd.sh

```diff
@@ -202,6 +202,14 @@ function collect_debug_info_after_kurl() {
     echo "";
 }
 
+function store_join_command() {
+    joincommand=$(curl -fsSL https://kurl.sh/version/v2022.04.08-1/latest/tasks.sh | sudo bash -s join_token ha)
```

Comment:
i didnt think of this before but we must run install with the `ha` flag [on line 129](https://github.com/replicatedhq/kURL/pull/2853/files#diff-500bcaf02f75d5650177d09cf6f73ade8068598454cbabfdd394218364340106R129) otherwise im not sure how the master join command will behave. you should add it and hopefully no other changes will be needed.